CONTRIBUTING.md

```diff
@@ -19,6 +19,9 @@ Please note that modifications should follow these coding guidelines:
 *   Code should pass the `eslint` linter
 *   Vertical whitespace helps readability, don’t be afraid to use it
 
+## Discussion
+Join the slack channel [linter-plugin-dev](https://atomio.slack.com/messages/linter-plugin-dev/) to discuss with fellow members.
```

Comment:
How about this?

>If you have any questions during development you can join the #linter-plugin-dev channel on [Atom's Slack instance](http://atom-slack.herokuapp.com/).

A direct link to the channel won't work as users need to already have joined Atom's Slack instance or that will just lead them to a sign in page. The link above is a sign in _and_ register page.